packages/components/src/components/Note/Note.module.css

```diff
@@ -0,0 +1,71 @@
+@import "@/styles";
+
+.root {
+  border-radius: var(--note--border-radius);
+  border: var(--note--border-width--default) var(--note--border-style)
+    var(--note--info-border-color);
+  border-inline-start-width: var(--note--border-width--main);
+  padding: var(--note--padding);
+  display: grid;
+  grid-template-columns: auto 1fr;
+  grid-column-gap: var(--note--icon-to-text-spacing);
+  grid-row-gap: var(--note--text-to-text-spacing);
+
+  /* Elements */
+
+  .customIcon,
+  .statusIcon {
+    grid-area: icon;
+    color: var(--note--info-heading-color);
+    font-size: var(--note--icon-size);
+    line-height: calc(var(--note--heading-font-size) * 1.5);
+  }
+
+  .heading {
+    grid-area: heading;
+    color: var(--note--info-heading-color);
+    font-size: var(--note--heading-font-size);
+  }
+
+  .content {
+    grid-area: content;
+  }
+
+  /* With custom icon */
+
+  &:has(.customIcon) .statusIcon {
+    display: none;
+  }
+
+  /* Without content */
+
+  &:not(:has(.content)) {
```

Comment:
Ist die Unterscheidung ob mit/ohne Content notwendig, da ja bei "ohne Content" die Area einfach nicht befüllt wird?

Reply:
![Bildschirmfoto 2024-01-11 um 11 31 43](https://github.com/mittwald/flow/assets/84317589/ff731939-ec6d-48ba-9f74-6820d599f94d)

Es wird zwar nicht beüllt, aber die Gap ist trotzdem da